### DISCUSSION TITLE
MNIST classification example

### DISCUSSION THREAD

**Discussion by Pacifictrip** [deepjavalibrary/djl#911] [Q&A]
Hello,

The MNIST Digit characters example works great, so I wonder how to configure Mnist Class to make the job with the EMNIST Letters characters instead of digits.
Is it feasible using Mnist Class or should I implement new one ?

Thank you !

**Comment by frankfliu** [comment]
You can create your own dataset, see: https://github.com/deepjavalibrary/djl/blob/master/basicdataset/src/main/java/ai/djl/basicdataset/cv/classification/FashionMnist.java

If you want to contribute to DJL, I can help you upload the required artifacts to DJL's S3 bucket.
